src/NServiceBus.AcceptanceTests/Core/OpenTelemetry/When_sending_messages_with_custom_activity.cs

```diff
@@ -0,0 +1,95 @@
+namespace NServiceBus.AcceptanceTests.Core.OpenTelemetry
+{
+    using System;
+    using System.Collections.Immutable;
+    using System.Diagnostics;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using NServiceBus.AcceptanceTesting;
+    using NServiceBus.AcceptanceTests.EndpointTemplates;
+    using NServiceBus.Pipeline;
+    using NUnit.Framework;
+
+    public class When_sending_messages_with_custom_activity : OpenTelemetryAcceptanceTest
```

Comment:
should this acceptance test be covered by unit tests instead? This seems to be very focused on the implementation details.

Reply:
We discussed this and we agreed that moving to a unit test is warranted